src/coreclr/inc/readytorun.h

```diff
@@ -445,6 +447,8 @@ enum ReadyToRunHelper
     // Array helpers for use with native ints
     READYTORUN_HELPER_Stelem_Ref_I                = 0x113,
     READYTORUN_HELPER_Ldelema_Ref_I               = 0x114,
+
+    READYTORUN_HELPER_NativeMemSet                = 0x115,
```

Comment:
We may want to rename the existing MemSet to NativeMemSet in case there are R2R images out there that use MemSet in the method prolog where the new MemSet does not work, and give a new ID to the new MemSet (it can be 0x3F - right before the current MemSet).


Some existing R2R image

Reply:
Addressed